config/schema/schema.config

```
@@ -56,6 +56,7 @@ schema {
     */
     check_path = { String p, String mode ->
         def File file = new File(p)
+        file = new File(file.getAbsolutePath())
```

```suggestion
        file = file.getAbsoluteFile()
```
From [the docs](https://docs.oracle.com/javase/8/docs/api/java/io/File.html#getAbsoluteFile--):
> Returns the absolute form of this abstract pathname. Equivalent to `new File(this.getAbsolutePath()`

---

Ah that's better. `.getAbsolutePath` returns a string.